etc/kayobe/stackhpc.yml

```diff
@@ -130,6 +130,23 @@ stackhpc_kolla_source_version: "stackhpc/{{ openstack_release }}"
 stackhpc_kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"
 stackhpc_kolla_ansible_source_version: "stackhpc/{{ openstack_release }}"
 
+###############################################################################
+# Overcloud host image source
+
+stackhpc_download_overcloud_images: false
```

Comment:
Without a checksum for these we're going to end up downloading them every time. Bifrost doesn't support providing a checksum for the cirros image. That's probably ok for a first pass, but we might want to revisit it.